packages/core/src/App/Containers/Redirect/redirect.jsx

```diff
@@ -59,10 +59,13 @@ const Redirect = ({
             });
             break;
         }
-        case 'mt5_password_reset':
-            history.push(`${routes.mt5}?code=${url_params.get('code')}#reset-password`);
+        case 'mt5_password_reset': {
+            localStorage.setItem('mt5_reset_password_code', url_params.get('code'));
```

Comment:
@salar-fs , would this work if the reset link goes to a different browser? 
Because if we're going to use `localStorage` it wouldn't sync if the user redirects / copy pastes the link to another browser from where he initial set the reset password.